include/gsl/span

```diff
@@ -121,139 +122,165 @@ namespace details
     {
     };
 
-    template <class Span, bool IsConst>
+    template <class type>
```

Comment:
Don't template parameters in this codebase conventionally have PascalCase names? (And on 224 and 238 and... many other places ;).)

Reply:
Why yes... yes they do. Will fix that. Thanks